tests/configs/modules/calendar/auth-default.js

```diff
@@ -26,7 +26,7 @@ var config = {
 				calendars: [
 					{
 						maximumNumberOfDays: 10000,
-						url: "http://localhost:8011/tests/configs/data/calendar_test.ics",
+						url: "http://localhost:8080/tests/configs/data/calendar_test.ics",
 						auth: {
```

Comment:
the ports must also changed in the tests ...

Reply:
Fixed those. not sure why they were in there, so I removed them where they didnt seem needed...